.github/workflows/ci.yml

```
@@ -0,0 +1,25 @@
+name: ci
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  vexchange-api-ci:
+    name: Vexchange API - CI
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: 📦 installing packages
+        run: npm install
```

`npm install` -> `npm ci`
```suggestion
        run: npm ci
```

---

Look here as a reference: https://github.com/proximacapital/typescript-template/blob/prod/.github/workflows/build-lint-test.yml